### PR TITLE
=pro Fix unmaintained scalariform github repo link.

### PR DESCRIPTION
**Pull request by szymon-rd**

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

**Comment by johanandren** [comment]
LGTM, thanks!

**Comment by johanandren** [comment]
OK TO TEST

**Comment by ktoso** [comment]
LGTM (added @johanandren to admin whitelist)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ the validator to test all projects.
 
 ## Source style
 
-Akka uses [Scalariform](https://github.com/mdr/scalariform) to enforce some of the code style rules.
+Akka uses [Scalariform](https://github.com/daniel-trinh/scalariform) to enforce some of the code style rules.
```
Comment (patriknw) [inline CONTRIBUTING.md]: technically speeking we have not updated to that plugin yet, but doesn't matter

Comment (ktoso) [inline CONTRIBUTING.md]: Yeah, though the same. Will actually ticketify upgrading

**Pull request merged**